kubectl-minio/README.md

```diff
@@ -0,0 +1,71 @@
+# MinIO Kubectl Plugin
+
+This is a `kubectl` plugin to interact with MinIO Operator on Kubernetes.
+
+## Prerequisites
+
+- Kubernetes cluster with storage nodes earmarked.
+- Required number of drives attached to each storage node. All drives mounted and formatted.
+- kubectl installed on your local machine, configured to talk to the Kubernetes cluster.
+
+Common Flags
+
+- `--namespace=minio-operator`
+- `--kubeconfig=`
+
+## Operator Deployment
+
+Command:
+
+`kubectl minio init [options] [flags]`
+
+Options:
+
+- `--image=minio/k8s-operator:2.0.6`
+- `--namespace-to-watch=default`
```

Comment:
Operator should work across all namespaces?

Reply:
> Maybe a comma-separated list?

Currently the operator can either watch all namespaces or one namespace set in this `WATCH_NAMESPACE` env.